classes/Employee.php

```diff
@@ -713,4 +713,17 @@ public function can($action, $tab)
 
         return ($access[$action] == '1');
     }
+    
+    /**
+     * Returns the default tab class name.
+     *
+     * @return null|string
+     */
+    public function getDefaultTabClassName()
```

Comment:
I don't see why you add this, in the context of Dispatcher refactoring : what I'm missing here? :thinking:

Reply:
@mickaelandrieu It's meant to get the default tab name from the model ($employee->getDefaultTabClassName()) inside the Dispatcher::setDefaultController() method